manifests/config/resource_source.pp

```diff
@@ -93,6 +95,7 @@
   Integer $url_timeout                                           = $rundeck::params::url_timeout,
   Boolean $use_default_mapping                                   = true,
   Optional[String] $endpoint_url                                 = undef,
+  Optional[String] $assume_role_arn                              = undef,
```

Comment:
Can you please enforce the minimal string length?
```suggestion
  Optional[String[1]] $assume_role_arn                           = undef,
```

Reply:
Done